protocol/0090-VAMM-automated_market_maker.md

```diff
@@ -127,6 +127,18 @@ In addition to amending to reduce the size a user may also cancel their AMM enti
 
 Note that, whilst an `Abandon Position` transaction immediately closes the AMM a `Reduce-Only` transaction will keep it active for an uncertain amount of time into the future. During this time, any Amendment transactions received should move the AMM out of `Reduce-Only` mode and back into standard continuous operation.
 
+### Oracle-driven base price
+
+The base price can either be a fixed scalar specified by a user or an [oracle configuration](./0045-DSRC-data_sourcing.md). It should be possible to switch between the two with a regular vAMM amendment transaction.
+
+When oracle-driven base price is selected during the vAMM creation then the system first awaits the receipt of the next data point from the oracle and then follows the process outlined in [creation/amendment process](#creationamendment-process) with `base price` set to that value.
+
+Similarly, when the vAMM configuration is amended to use the oracle-driven base price rather than a fixed user supplied value, the system awaits the receipt of the next data point from the oracle and then uses that value as the `base price` in the regular amendment transaction outlined above. Please note that **configuration amendment** from a fixed to oracle-driven base price should only be rejected if the oracle specification is incorrect. If the `base price` received from the oracle is such that the **value amendment** is rejected then the value should remain unchanged, but the next **value amendment** should still be attempted the next time around.
+
+#### Maximum update frequency
+
+An optional non-negative parameter controlling the maximum update frequency should be included in the oracle-driven base price configuration. When set to `0` the `base price` will update each time a new value is received from the oracle. When set to a positive value, the `base price` should be amended to the last value received from the oracle once the specified time elapses. If such value is the same as the currently specified `base price` then there should be no effect. The default value of this parameter should be `0`.
```

Comment:
This is a bit confusing, we have 'frequency' but then say the update happens after a time interval elapses. Should we call it "Maximum update time interval"?

Reply:
Maybe I'm missing something, but I don't think we even need this field since I think it can be handled solely by the existing data source framework we have. 

For example if we set it up an Ethereum contract call to drive the base price, there is already a field in the data-source definition for "how frequently to read the contract". And if you are sending in new prices by hand using a normal trusted-key oracle-submission-transaction then you'll just only send it in when you want it updated anyway.

Reply:
Would it be better to have % change of how far out the base price is from the oracle input?

That's to avoid paying taker fees when changes to base price would result in trades for the vAMM.

Reply:
Modified the spec to use the relative price change.